SpatialGDK/Source/SpatialGDK/Public/Interop/ActorSystem.h

```diff
@@ -18,6 +20,9 @@ class USpatialNetDriver;
 class SpatialActorChannel;
 class USpatialNetDriver;
 
+using FChannelsToUpdatePosition =
```

Comment:
The coding standard suggests against using `using` declarations at global scope: https://docs.unrealengine.com/en-US/ProductionPipelines/DevelopmentSetup/CodingStandard/#namespaces

> * Do not put using declarations in the global scope, even in a .cpp file (it will cause problems with our "unity" build system.)

Reply:
Also, I wonder about the KeyFuncs passing `false` to `bInAllowDuplicateKeys` since we seem to always use it - can we specify it as a default argument? Isn't related to the immediate PR, of course.

Reply:
great catch- moved within SpatialGDK

Reply:
For the other comment, I think it makes sense, and it's very small, so added as a drive by.